### PR TITLE
docs(npm): Fix NPM scope link

### PR DESCRIPTION
**Pull request by strogonoff**
npmjs.com/~material leads nowhere and it’s not immediately obvious that ~material-admin is the correct scope.

### CODE REVIEW

**Comment by strogonoff** [comment]
@googlebot I signed it!

**Comment by strogonoff** [comment]
> FWIW we also filed an issue with NPM to auto-redirect URLs where users have been migrated to orgs

Awesome! Working link is good though, I was trying to get a sense of this project’s infrastructure before adopting the dependency and perhaps I’m not alone :)

**Comment by traviskaufman** [comment]
@strogonoff looks like our CLA Bot didn't pick up your comment. Could you reply with just:

> I signed it!

And see if that works. If not, I can try some machinery on my end to fix. We need the CLABot to go green before we can merge. Sorry!

**Comment by strogonoff** [comment]
I signed it!

**Comment by strogonoff** [comment]
@traviskaufman Thanks for the heads-up.

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ introduction to the library.
 ## Installing individual components
 
 MDC-Web is modular by design. Each component lives within its own packages under the
-[@material npm scope](https://www.npmjs.com/~material).
+[@material-admin npm scope](https://www.npmjs.com/~material-admin).
```
Comment (traviskaufman) [inline README.md]: Went back over this after reading @amsheehan's comment. What it should read is:

```md
[@material npm org](https://www.npmjs.com/org/material)
```

Comment (amsheehan) [inline README.md]: Agreed!

**Comment by amsheehan** [comment]
@strogonoff ~bump~

**Comment by strogonoff** [comment]
@amsheehan I was under the impression that this request is to be closed without merging. If still relevant, I’ve just updated it so that README link points to https://www.npmjs.com/org/material. Cheers!

**Pull request merged**